kubetest/e2e.go

```diff
@@ -603,35 +603,46 @@ func kubemarkUp(dump string, o options, deploy deployer) error {
 		return err
 	}
 
-	masterIP, err := control.Output(exec.Command(
+	masterIPs, err := control.Output(exec.Command(
 		"gcloud", "compute", "addresses", "describe",
 		os.Getenv("MASTER_NAME")+"-ip",
 		"--project="+o.gcpProject,
 		"--region="+o.gcpZone[:len(o.gcpZone)-2],
 		"--format=value(address)"))
 	if err != nil {
-		return fmt.Errorf("failed to get masterIP: %v", err)
+		return fmt.Errorf("failed to get masterIPs: %v", err)
 	}
-	if err := os.Setenv("KUBE_MASTER_IP", strings.TrimSpace(string(masterIP))); err != nil {
+	if err := os.Setenv("KUBE_MASTER_IP", strings.TrimSpace(string(masterIPs))); err != nil {
 		return err
 	}
+	// Leaving this temporarily until https://github.com/kubernetes/perf-tests/issues/595 is closed
 	// MASTER_IP variable is required by the clusterloader. It requires to have master ip provided,
 	// due to master being unregistered.
-	if err := os.Setenv("MASTER_IP", strings.TrimSpace(string(masterIP))); err != nil {
+	if err := os.Setenv("MASTER_IP", strings.TrimSpace(string(masterIPs))); err != nil {
+		return err
+	}
+	// MASTER_IPS variable is required by the clusterloader. It requires to have master ip(s) provided,
```

Comment:
I think this is no longer true in kubemark (masters not being registered). Given that, I think we can get rid of all this logic, as clusterloader2 should be able to obtain master info (including internal / public ips) from the node list.

Adding @mborsz, who implemented HA support in kubemark and will be able to confirm or correct me.

Reply:
Alright if that's the case I'll remove it. Was I working on the wrong stuff you proposed in https://github.com/kubernetes/perf-tests/issues/595?

Reply:
That was my mistake, I wasn't aware that these envs are set only in one place and only for kubemark.
I think kubemark doesn't require them anymore, but let's wait for @mborsz to confirm.

Reply:
Yes, the env variables here are no longer needed for kubemark (and any other cluster with master registered).